src/SME.SR.Data/Repositories/Sgp/PendenciaRepository.cs

```diff
@@ -174,16 +182,135 @@ public PendenciaRepository(VariaveisAmbiente variaveisAmbiente)
 
             if (bimestre > 0)
                 query.AppendLine($" and pe.bimestre  = @bimestre");
+
+            query.AppendLine(" union all  ");
+            query.AppendLine(@" select distinct 
+                            p.titulo,
+	                        p.descricao as Descricao,
+	                        p.situacao,
+                            p.instrucao, 
+	                        d.abreviacao as DreNome,
+	                        te.descricao || ' - ' || u.nome as UeNome,
+	                        t.ano_letivo as AnoLetivo,
+	                        t.modalidade_codigo as ModalidadeCodigo,
+	                        t.semestre,
+	                        t.nome || ' - ' || t.ano || 'ºANO' as TurmaNome,
+                            t.turma_id as TurmaCodigo,
+	                        a.disciplina_id::bigint as DisciplinaId,
+	                        pe.bimestre,
+                            usu.nome as criador,
+                            usu.login as criadorRf,
+	                        p.alterado_por as aprovador,
+	                        p.alterado_rf as aprovadorRf,
+                            'Calendário' as TipoPendencia,
+                            true as OutrasPendencias,
```

Comment:
pendencia_aula tem turma e componente então :
```suggestion
                            false as OutrasPendencias,
```